pgjdbc/src/main/java/org/postgresql/ds/common/BaseDataSource.java

```diff
@@ -519,6 +519,22 @@ public void setSslMode(String mode) {
   }
 
   /**
+   * @return SSL Algo Used
+   * @see PGProperty#SSL_FACTORY_PROTO
+   */
+  public String getSslFactoryAlgo() {
+    return PGProperty.SSL_CONTEXT_PROTOCOL.get(properties);
+  }
+
+  /**
+   * @param arg argument forwarded to SSL Context
+   * @see PGProperty#SSL_FACTORY_PROTO
+   */
+  public void setSslFactoryAlgo(String arg) {
```

Comment:
I could not find anyplace where any of the getters/setters in this class were used (not just the two I added) but I added a getter/setter for my new property just to be consistent. However I should change the name to match the updated property name.